### PR TITLE
start to build inbox as designed, in angular interface

### PR DESCRIPTION
**Pull request by robguthrie**

### CODE REVIEW

File: lineman/app/modules/inbox/inbox.coffee [hunk 1]
```diff
@@ -3,6 +3,19 @@ angular.module('loomioApp').directive 'inbox', ->
...
-    $scope.currentUser = UserAuthService.currentUser
-    Records.discussions.fetchInbox()
+  controller: ($scope, InboxService) ->
+    # build a loki view for each of these modes.
+    # maybe within the service
+    InboxService.fetchRecords().then (data) ->
+      $scope.unreadThreads = InboxService.unreadThreads()
+      $scope.currentThreads = InboxService.currentThreads()
+
+    $scope.unreadThreads = InboxService.unreadThreads()
```
Comment (gdpelican) [inline lineman/app/modules/inbox/inbox.coffee]: I'm confused by this; why call these twice? And why pass data through the fetchRecords promise?

File: lineman/app/services/inbox_service.coffee [hunk 2]
```diff
@@ -0,0 +1,34 @@
+angular.module('loomioApp').factory 'InboxService', (UserAuthService, Records) ->
+  new class InboxService
+    constructor: ->
+      currentUser = UserAuthService.currentUser
+      baseThreadsView = (name) ->
+       view = Records.discussions.collection.addDynamicView(name)
```
Comment (gdpelican) [inline lineman/app/services/inbox_service.coffee]: Should clean this up before merge

Comment (robguthrie) [inline lineman/app/services/inbox_service.coffee]: hey james.. looks like this isnt really ready for review.. thanks for the
feedback but know that it's not at that stage yet

Comment (gdpelican) [inline lineman/app/services/inbox_service.coffee]: Oh, sorry!

I at one point for some reason thought this was trying to go in before the runthrough today.

Consider my jets cooled :snowflake:

**Pull request merged**